mne/time_frequency/tests/test_tfr.py

```diff
@@ -118,9 +115,6 @@ def test_time_frequency():
     epochs_power_2 = abs(epochs_power_complex)
     epochs_power_3 = epochs_power_2.copy()
     epochs_power_3.data[:] = np.inf  # test that it's actually copied
-    assert_array_almost_equal(epochs_power_2.data, epochs_power_picks.data)
```

Comment:
Instead of killing these, if they should check the opposite, you could do something like:
```
assert not np.allclose(..., atol=1-20)
```
Or whatever makes the point that they actually shouldn't be equal